recipes/orekit/test/EventDetectorTest.py

```diff
@@ -0,0 +1,129 @@
+# -*- coding: utf-8 -*-
```

Comment:
Is this already available from the source? If so, would use [`source_files`]( https://conda.io/docs/user-guide/tasks/build-packages/define-metadata.html#source-files ) instead of including it in the recipe.

Reply:
fixed